tests/acquia_lift.test

```diff
@@ -3290,7 +3290,7 @@ class AcquiaLiftWebTestCampaignWizard extends AcquiaLiftWebTestBase {
     // Try to start the agent.
     $this->drupalPost('admin/structure/personalize/manage/' . $agent->machine_name . '/review', array(), $this->getButton('wizard_start'));
     $this->assertText('No goals have been set up for agent ' . $agent->machine_name);
-    $this->assertText('There was a problem syncing your personalization components to Lift and it cannot be started at this time.');
```

Comment:
We could still check for the error message: No goals have been set up for agent $agent->machine_name

Speaking of that, however, since we are just throwing the error messages directly, we should clean "agent" out of them as it is no longer on the front-end.  Could we also provide a link to the review page for the campaign/personalization so that the user can see more clearly what the problem is along with the "Let's fix this" text.